src/controllers/open-data-store.js

```diff
@@ -0,0 +1,17 @@
+const storeAll = require('../utils/store-odp-data')
+
+/**
+ * Starts the async function to obtain and store
+ * the dataset data into the Redis cache
+ */
+module.exports = async function(req, res) {
```

Comment:
This would be a security nightmare 😈 

Baddies could swap out our datasets at will. I think you should be able to do without this file.

Reply:
Removed this functionality